packages/nns/src/governance.canister.ts

```diff
@@ -462,6 +464,41 @@ export class GovernanceCanister {
     });
   };
 
+  /**
+   * Simulate merging two neurons
+   *
+   * @throws {@link GovernanceError}
+   */
+  public simulateMergeNeurons = async (request: {
+    sourceNeuronId: NeuronId;
+    targetNeuronId: NeuronId;
+  }): Promise<NeuronInfo> => {
+    const rawRequest = toMergeRequest(request);
+
+    const command = await simulateManageNeuron({
+      request: rawRequest,
+      service: this.certifiedService,
+    });
+
+    if (
+      "Merge" in command &&
+      command.Merge.target_neuron_info[0] !== undefined &&
```

Comment:
I haven't tried. I'm following the pattern from `spawnNeuron`, `splitNeuron`, `disburse` and `spawnHardwareWallet`. Shall I make a PR to change all of them?

Reply:
Ah ok. I guess it's because we introduced the utils afterwards.
Not a must. Of course appreciated if you used for new piece of code.
Thanks for the feedback.